backend/src/schema/types/type/NOTIFIED.gql

```diff
@@ -0,0 +1,28 @@
+type NOTIFIED {
+  from: NotificationSource
+  to: User
+  createdAt: String
+  read: Boolean
+  reason: NotificationReason
+}
+
+union NotificationSource = Post | Comment
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T08:30:50Z" title="Friday, August 30th 2019, 10:30:50 am +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,28 @@
+type NOTIFIED {
+  from: NotificationSource
+  to: User
+  createdAt: String
+  read: Boolean
+  reason: NotificationReason
+}
+
+union NotificationSource = Post | Comment
```


Yeah 😍

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:30:33Z" title="Friday, August 30th 2019, 2:30:33 pm +02:00">Aug 30, 2019</time>_

---


we learn new exciting things every day!!